.github/workflows/markdown-link-check.yml

```
@@ -0,0 +1,31 @@
+name: Markdown Link Check
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    paths:
+      - "**/*.md"
```

```suggestion
      - "**.md"
```
Also remove all other files from this list.